shipment_advice_planner_toursolver/views/toursolver_resource.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="toursolver_resource_form_view">
+        <field name="model">toursolver.resource</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <label for="name" class="oe_edit_only" />
+                      <h1>
+                        <field name="name" class="oe_inline" />
+                      </h1>
+                    <group>
+                        <field name="toursolver_backend_id" />
+
+                    </group>
+                    <group>
+
+                        <group>
+                        <field name="resource_id" />
+                        <field name="use_delivery_person_coordinates_as_end" />
+
+                        </group>
+                        <group>
+                        <field
+                                name="partner_id"
+                                attrs="{'required': [('use_delivery_person_coordinates_as_end', '=', True)]}"
+                            />
+
+                        </group>
+                        <field name="resource_properties" />
```

Comment:
@sbejaoui I would be usefull to add a text explaining what properties are, how to use them and where to find the possible names (direct link to the doc)